TAF/config/global_variables.py

```diff
@@ -2,8 +2,8 @@
 
 LOG_LEVEL = "INFO"
 
-# Version in the Consul path for all services
-CONSUL_CONFIG_VERSION = "2.0"
+# Base endpoint in the Consul path for all services
+CONSUL_CONFIG_BASE_ENDPOINT = /v1/kv/edgex/v3
```

Comment:
```suggestion
CONSUL_CONFIG_BASE_ENDPOINT = "/v1/kv/edgex/v3"
```

Reply:
https://github.com/edgexfoundry/edgex-taf/blob/8759bd2e28b253ee0e7a88d10971ac5d5af214f2/TAF/testScenarios/integrationTest/UC_metadata_notifications/metadata_notifications.robot#L101
This file also needs to be updated.

Reply:
@cherrycl , fixed